modules/metastorage-client/src/integrationTest/java/org/apache/ignite/internal/metastorage/client/ITMetaStorageServiceTest.java

```diff
@@ -181,8 +179,18 @@ public void beforeTest() {
         var nodeFinder = new LocalPortRangeNodeFinder(NODE_PORT_BASE, NODE_PORT_BASE + NODES);
 
         nodeFinder.findNodes().stream()
-            .map(addr -> startClusterNode(addr, nodeFinder))
-            .forEach(cluster::add);
+            .map(addr -> ClusterServiceTestUtils.clusterService(
+                addr.toString(),
```

Comment:
Not sure, but seems like indentation is missing

Reply:
Fixed.